index.html

```diff
@@ -100,60 +100,66 @@ <h3 class="article__title">How can I help?</h3>
                     Learn more
                   </a>
                 </p>
-              </div>
+              </article>
             </div>
           </div>
-        </div>
+        </main>
 
-        <div class="sidebar">
+        <aside class="sidebar">
           <h2 class="heading-underline">Upcoming events</h2>
 
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
-              </h3>
-              <p class="article__summary">
-                Riders needed now to help raise awareness and funds for BfR and
-                Médecins Sans Frontières.
-              </p>
+          <article class="article">
+            <div class="articles"> 
```

Comment:
You have this the other way round on lines 74-75. Update the `<div>` to a `<section>` which contains the `<article>` tag. 
Check if there's any other places in your html where you could change this.